cmd/resim/commands/batch.go

```diff
@@ -33,15 +33,19 @@ var (
 		Run:   getBatch,
 	}
 
+	buildIDString          string
+	experienceIDsString    string
+	experienceTagIDsString string
+
 	batchIDString string
 	batchName     string
 	exitStatus    bool
 )
 
 func init() {
-	createBatchCmd.Flags().String("build_id", "", "The ID of the build.")
-	createBatchCmd.Flags().String("experience_ids", "", "Comma-separated list of experience ids to run.")
-	createBatchCmd.Flags().String("experience_tag_ids", "", "Comma-separated list of experience tag ids to run.")
+	createBatchCmd.Flags().StringVar(&buildIDString, "build_id", "", "The ID of the build.")
+	createBatchCmd.Flags().StringVar(&experienceIDsString, "experience_ids", "", "Comma-separated list of experience ids to run.")
+	createBatchCmd.Flags().StringVar(&experienceTagIDsString, "experience_tag_ids", "", "Comma-separated list of experience tag ids to run.")
```

Comment:
just realized that we will probably need to get a --gitlab option for this, but we can do this later